composer.json

```diff
@@ -87,7 +88,7 @@
 			"@php ./vendor/bin/phpunit --no-coverage --order-by=random"
 		],
 	   	"static-analysis": [
-			"@php ./vendor/bin/phpstan analyze --memory-limit 536870912"
+			"@php ./vendor/bin/phpstan analyze --memory-limit 1074000000"
```

Comment:
If we don't want any memory limit then we can do following which I think is better

```
@php ./vendor/bin/phpstan analyze --memory-limit=-1
```

Reply:
I'll leave as-is, but we can revisit this if needed. Good tip by the way!